src/Dockerfile

```diff
@@ -13,7 +13,7 @@ RUN echo "deb [signed-by=/usr/share/keyrings/ria-repository.gpg] https://install
     apt-get clean && \
     rm -rf /var/lib/apt/lists
 
-COPY ./WebEid.AspNetCore.Example/bin/Release/net6.0/publish/ .
+COPY ./WebEid.AspNetCore.Example/bin/x64/Release/net6.0/publish/ .
```

Comment:
Is there any reason why we cannot use AnyCPU and have to use the x64 build?

Reply:
Like I said, RIA provides only x64 binaries on windows and ubuntu.
Most mac developers have arm64 computers, therefore they cannot run/debug/develop the application on windows and ubuntu.